openfisca_us/variables/irs/outputs.py

```diff
@@ -875,11 +875,127 @@ def formula(tax_unit, period, parameters):
+    documentation = "Whether this individual has retired on disability, and was permanently and totally disabled when they retired"
+    definition_period = YEAR
+
+    # Definition from 26 U.S. Code § 22 - Credit for the elderly and the permanently
+    # and totally disabled:
+
+    # An individual is permanently and totally disabled if he is unable to engage
+    # in any substantial gainful activity by reason of any medically determinable
+    # physical or mental impairment which can be expected to result in death or
+    # which has lasted or can be expected to last for a continuous period of not
+    # less than 12 months. An individual shall not be considered to be permanently
+    # and totally disabled unless he furnishes proof of the existence thereof in
+    # such form and manner, and at such times, as the Secretary may require.
+
+
+class qualifies_for_elderly_or_disabled_credit(Variable):
+    value_type = bool
+    entity = Person
+    label = "Qualifies for elderly or disabled credit"
+    documentation = (
+        "Whether this tax unit qualifies for the elderly or disabled credit"
+    )
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        elderly_disabled = parameters(period).irs.credits.elderly_or_disabled
+        return person("age", period) >= elderly_disabled.age | person(
+            "is_retired_on_total_disability", period
+        )
+
+
+class total_disability_payments(Variable):
+    value_type = float
+    entity = Person
+    label = "Disability (total) payments"
+    unit = "currency-USD"
+    documentation = "Wages (or payments in lieu thereof) paid to an individual for permanent and total disability"
+    definition_period = YEAR
+
+
+class section_22_income(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Section 22 income"
+    unit = "currency-USD"
+    documentation = (
+        "Income upon which the elderly or disabled credit is applied"
+    )
+    definition_period = YEAR
+
+    def formula(tax_unit, period, parameters):
+        elderly_disabled = parameters(period).irs.credits.elderly_or_disabled
+        # Calculate initial amount
+        mars = tax_unit("mars", period)
+        num_qualifying_individuals = tax_unit.sum(
+            tax_unit.members(
+                "qualifies_for_elderly_or_disabled_credit", period
+            )
+        )
+        initial_amount = select(
+            [
+                num_qualifying_individuals == 1,
+                num_qualifying_individuals == 2,
+                mars == mars.possible_values.SEPARATE,
+                True,
+            ],
+            [
+                elderly_disabled.amount.one_qualified,
+                elderly_disabled.amount.two_qualified,
+                elderly_disabled.amount.separate,
+                0,
+            ],
+        )
+
+        # Limitations on under-65s
+
+        is_over_age = tax_unit.members("age", period) >= elderly_disabled.age
```

Comment:
Avoids the `over` / `>=` issue
```suggestion
        is_elderly = tax_unit.members("age", period) >= elderly_disabled.age
```